.travis.yml

```diff
@@ -1,27 +1,32 @@
-# make it explicit that we favor the new container-based travis workers
+# Make it explicit that we favor the
+# new container-based Travis workers
 language: python
+dist: xenial
 
 cache:
   apt: true
   directories:
   - $HOME/.cache/pip
   - $HOME/.ccache
 
-dist: xenial
-
 env:
   global:
     # Directory where tests are run from
     - TEST_DIR=/tmp/sklearn
     - OMP_NUM_THREADS=2
     - OPENBLAS_NUM_THREADS=2
+    - SKLEARN_SKIP_NETWORK_TESTS=1
```

Comment:
Those variables probably need to be passed to the docker build and test environment via `CIBW_ENVIRONMENT` instead.